### ISSUE TITLE
Example of CustomPathResolver doesn't work

### ISSUE THREAD

**Issue by dobau** [caelum/vraptor4#991]
The implementation example for create a CustomPathResolver doesn't work.

Example from http://www.vraptor.org/pt/docs/componentes/.

Is it missing of the Inject annotation, right?

    @Inject
    public CustomDefaultPathResolver(FormatResolver resolver) {
        super(resolver);
    }

**Comment by garcia-jj** [comment]
Yep, is missing. You need something like this:

```java
@ApplicationScoped
@Specializes
@Priority(Interceptor.Priority.APPLICATION + 10)
```

**Comment by Turini** [comment]
hi @dobau, you're right. It's missing the delegate constructor. It'd look like this:
```java
@Specializes
public class CustomPathResolver extends DefaultPathResolver {
	/** 
	 * @deprecated CDI eyes only
	 */
	protected CustomPathResolver() {
		this(null);
	}

	@Inject
	public CustomPathResolver(FormatResolver resolver) {
		super(resolver);
	}
	// ...
}
```
No need to add `@priority`, since it's not an alternative. 
Would you like to send us a pull request with this change? It'll be very welcome :)

**Comment by Turini** [comment]
Closed by #993
